cli/src/semgrep/commands/ci.py

```diff
@@ -223,14 +223,23 @@ def ci(
     state.error_handler.configure(suppress_errors)
     scan_handler = None
 
+    if allow_untrusted_validators and not run_secrets_flag:
+        logger.debug(
+            "Implicity enable secrets validation since allow_untrusted_validators is set."
+        )
+        run_secrets_flag = True
+
     token = state.app_session.token
     if not token and not config:
         # Not logged in and no explicit config
         logger.info("run `semgrep login` before using `semgrep ci` or set `--config`")
         sys.exit(INVALID_API_KEY_EXIT_CODE)
     elif not token and config:
         # Not logged in but has explicit config
-        pass
+        if run_secrets_flag:
```

Comment:
Thoughts on exiting with a message instead of just disabling and continuing the run?

Reply:
The idea here was to sorta do what the playground does and still run the rules without any validation. Basically keep this:
```
┌───────────────────────────────┐
│ 1 Unvalidated Secrets Finding │
└───────────────────────────────┘
```

But I'm open to just refusing to run any rules with a validator. Any thoughts here @leifdreizler or @LewisArdern?